libclambcc/ClamBCAnalyzer/ClamBCAnalyzer.h

```diff
@@ -24,15 +24,20 @@
 
 #include "Common/clambc.h"
```

Comment:
I think we can change this to just `#include "clambc.h" if we make the `Common` library a dependency of these other opt passes.

Reply:
Oh there are a bunch of other places where we `#include "Common/something"`, like:
```
#include "Common/ClamBCCommon.h"
#include "Common/ClamBCUtilities.h"
```

We should be able to simplify all of them to remove the `Common/` once we make `Common` a library dependency in the CMakeLists.txt's.

Reply:
done